lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -393,9 +393,6 @@ def _call(self, method: Callable, json_data: JSON):
         except KeyError as e:
             self.logger.exception("{} problem in preprocess, missing {}", klasname, e)
             abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
-        except Exception as e:
-            self.logger.exception("{} preprocess failed: {}", klasname, e)
-            abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
```

Comment:
Basically, since we're removing this from the inner API layers, we should have it at the outer layer.